hubspot3/test/test_tickets.py

```diff
@@ -1,19 +1,27 @@
 """
 testing hubspot3.tickets
 """
+import json
+from unittest.mock import Mock
+
 import pytest
+
 from hubspot3.tickets import TicketsClient
 from hubspot3.test.globals import TEST_KEY
 
-
 TICKETS = TicketsClient(api_key=TEST_KEY)
-
-
 # since we need to have an id to submit and to attempt to get a ticket,
 # we need to be hacky here and fetch one upon loading this file.
 BASE_TICKET = TICKETS.get_all(limit=1)[0]
 
 
+@pytest.fixture
+def tickets_client(mock_connection):
```

Comment:
Are the lib owners using fixtures in the same file as the test specs? I'd rather have such utilities separated, but keep it that way if the codebase already follows this standard.

Reply:
I agree, but it is their standard...